react/src/functions.tsx

```diff
@@ -376,3 +376,17 @@ export const updateTableFilter = (
         }
     }
 };
+
+export const checkPipelineStatusChange = (fromState: PipelineStatus, toState: PipelineStatus) => {
+    const endStates = [PipelineStatus.COMPLETED, PipelineStatus.ERROR, PipelineStatus.CANCELLED];
+    return (
+        // End states cannot change to another state
+        !(fromState in endStates) &&
```

Comment:
This looks like a Python-JavaScript mixup :grin:. ` !(endStates.includes(fromState))` gave me the results I expected.